source/includes/_android.md.erb

```diff
@@ -207,6 +208,10 @@ Used to pass meta information about a Chatter. This can be useful for tracking i
 
 You can also set "meta fields" using XML. For this, you need to create a JSON file in the res/raw directory, and set the reference to the view declaration:
 
+### sensitiveMetaFields
+
+Similar to `metaFields` except the values are not stored in the database and will be deleted after 24 hours.
```

Comment:
I think this statement is a little confusing. Do you mean that sensitive variables are only stored for 24 hours? Or they are not stored at all?

Reply:
I copied this from the embed documentation. I found it confusing also but I trusted it since I got it from the embed documentation.